### PR TITLE
bump freebsd ssl

### PR DESCRIPTION
**Pull request by Emilgardis**

### CODE REVIEW

**Comment by Emilgardis** [comment]
Why do we need this again?

**Comment by Emilgardis** [comment]
bors try --target freebsd

File: docker/freebsd-extras.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ main() {
     local arch="${1}"
 
     local sqlite_ver=3.35.5_4,1 \
```
Comment (Jules-Bertholet) [inline docker/freebsd-extras.sh:9]:
```suggestion
    local sqlite_ver=3.37.2,1 \
```

**Comment by Emilgardis** [comment]
@reitermarkus do you know why we add sqlite and openssl to freebsd? Seems contradictory, since we opted to not include openssl in other targets

**Comment by Emilgardis** [comment]
bors try --target freebsd

**Comment by Emilgardis** [comment]
@cross-rs/maintainers anyone able to r+ this?

**Comment by Emilgardis** [comment]
bors try

**Comment by Emilgardis** [comment]
ping @cross-rs/maintainers :)

**Review by svenstaro — APPROVED** [review]
If it builds, it builds :)

**Pull request merged**